src/current/v25.2/architecture/transaction-layer.md

```diff
@@ -432,6 +432,45 @@ Despite their logical equivalence, the transaction coordinator now works as quic
 
 Additionally, when other transactions encounter a transaction in `STAGING` state, they check whether the staging transaction is still in progress by verifying that the transaction coordinator is still heartbeating that staging transaction's record. If the coordinator is still heartbeating the record, the other transactions will wait, on the theory that letting the coordinator update the transaction record with the result of the attempt to commit will be faster than going through the transaction status recovery process. This means that in practice, the transaction status recovery process is only used if the transaction coordinator dies due to an untimely crash.
 
+### Buffered writes
+
+{% include_cached new-in.html version="v25.2" %} Buffered Writes enhance transaction throughput and reduce operational cost by minimizing the number of round-trips between the client and the database during write operations.
+
+{% include feature-phases/preview.md %}
+
+Buffered Writes work by temporarily storing a transaction's writes on the client-side ([gateway node]({% link {{ page.version.version }}/architecture/sql-layer.md %}#gateway-node)) until the transaction [commits](#parallel-commits). This approach reduces redundant writes, minimizes [pipeline](#transaction-pipelining) stalls, and allows for passive use of the [1-phase commit (1PC) fast-path]({% link {{ page.version.version }}/ui-distributed-dashboard.md %}#kv-transactions). By deferring writes until commit time, the system can serve read-your-writes locally, resulting in further potential performance gains.
+
+Buffered Writes update the transaction flow so that it has the following properties:
+
+1. Write buffering: Instead of sending each write operation immediately to the [leaseholder]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder), writes are buffered on the client-side until the transaction commits. This reduces the number of write operations sent to the [Distribution layer]({% link {{ page.version.version }}/architecture/distribution-layer.md %}).
```

Comment:
For many transactions, the impact of this buffering is not to reduce the number of write operations but to issue that same number of write operations in a single batch rather than multiple batches. This is what the write batching bit gets at below so perhaps you could combine this point and the other point. Maybe something like:

> 1. Write buffering: Instead of sending each write operation immediate to the leaseholder, writes are now buffered on the client-side until the transaction commits. The writes can then be issued in a single batch, reducing the number of total requests required to complete the transaction.

Reply:
updated to use your text with  "client-side" changed to "gateway node "